src/main/java/it/gov/pagopa/payhub/activities/mapper/ingestionflow/receipt/PaymentsReporting2ReceiptMapper.java

```diff
@@ -12,25 +13,25 @@
  */
 @Service
 public class PaymentsReporting2ReceiptMapper {
-	public static final String CREDITOR_REFERENCE_ID_PREFIX = "3";
+	public static final String NOTICE_NUMBER_PREFIX = "3";
 	public static final String ANONYMOUS_PERSON = "ANONIMO";
 	public static final String ALIAS_TEMPLATE = "CODE_%s_" + ReceiptOriginType.PAYMENTS_REPORTING.getValue();
 	public static final String CHANNEL = "BATCH";
 	/** Taxonomy for general incomes */
 	public static final String DEFAULT_TRANSFER_CATEGORY = "9/0801100AP/";
 
-	public ReceiptWithAdditionalNodeDataDTO map2DummyReceipt(PaymentsReporting paymentsReporting, String fiscalCodePA) {
+	public ReceiptWithAdditionalNodeDataDTO map2DummyReceipt(PaymentsReporting paymentsReporting, Organization organization) {
 		return new ReceiptWithAdditionalNodeDataDTO()
 			.ingestionFlowFileId(paymentsReporting.getIngestionFlowFileId())
 			.receiptOrigin(ReceiptOriginType.PAYMENTS_REPORTING)
 			.paymentReceiptId(paymentsReporting.getIur())
-			.noticeNumber(paymentsReporting.getIuv())
+			.creditorReferenceId(paymentsReporting.getIuv())
 			.orgFiscalCode(paymentsReporting.getReceiverOrganizationCode())
 			.outcome(paymentsReporting.getPaymentOutcomeCode())
-			.creditorReferenceId(CREDITOR_REFERENCE_ID_PREFIX + paymentsReporting.getIuv())
+			.noticeNumber(NOTICE_NUMBER_PREFIX + paymentsReporting.getIuv())
```

Comment:
move this into the class it.gov.pagopa.payhub.activities.util.DebtPositionUtilities
creating a public static method `iuv2nav`
Let's rename the constant (defined in the utility class) into AUX_DIGIT instead